.github/workflows/release.yml

```
@@ -0,0 +1,28 @@
+name: Release
+on:
+  push:
+    tags:
+      - v**
```

From the [workflow syntax cheatsheet](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet):

```
**: Matches zero or more of any character.
```

Works for me, but is there a specific reason not to make this simply `"*"` ?
```suggestion
      - "*"
```

---

Was thinking we only want to run this on tags that are prefixed with `v` (releases).